setup.py

```diff
@@ -15,10 +15,10 @@
       install_requires=[
           'attrs==16.3.0',
           'pendulum==1.2.0',
-          'singer-python==5.3.1',
+          'pipelinewise-singer-python @ git+https://github.com/transferwise/pipelinewise-singer-python.git#egg=pipelinewise-singer-python',
```

Comment:
I'm installing directly from github as we don't have `pipelinewise-singer-python` yet on PyPi, will update this once we do.

Reply:
Updated.